bindings/Makefile

```diff
@@ -16,11 +17,28 @@ deps/$(CODEGEN_BIN):
 build/%: $(SWAGGER_SPEC) deps/$(CODEGEN_BIN)
 	mkdir -p build/$*
 	java -jar deps/$(CODEGEN_BIN) generate -i $(SWAGGER_SPEC) \
-		-l $* -o build/$*
+		-l $* -o build/$* -c config.json
```

Comment:
remove the `config.json` from the old typescript target, it can only cause problems.